README.md

```diff
@@ -86,6 +86,23 @@ php artisan passport:install
 Use your webserver of choice or the in php included dev server (`php artisan serve`) to boot the application.
 You should see the Träwelling homepage at http://localhost:8000.
 
+### Option 3: Local Development using [Nix](https://nixos.org/)
+
+Nix is a cross-platform package manager for Linux and macOS systems.
+It also provides per project development environments.
+There is a also a Linux Distribution called NixOS which builds on top of nix,
+but it's not required to use nix and this development environment.
+
+Our [nix flake](flake.nix) includes such an environment with a pre configured MySQL instance.
+
+If you want to use it:
+
```

Comment:
Previously, I relied heavily on Laravel's config manager to substitute the defaults and only wrote down the differences to the default in my `.env`. This doesn't really work if other applications also work with the `.env` file.

For newcomers, I'd like to point out that they need to copy the `.env` file.

```suggestion

- Copy the `.env.example` file to `.env`
```

I have later seen that the script does that for you on the first run, so __I'm open for debate__ if we should double the instruction here.

Reply:
a note that the nix environment does that automatically would be an idea maybe